jujugui/static/gui/src/app/views/utils.js

```diff
@@ -2252,6 +2252,102 @@ YUI.add('juju-view-utils', function(Y) {
     return xhr;
   };
 
+  /**
+    Deploy or commit to a model.
+
+    @method deploy
+    @param {Object} env Reference to the app env.
+    @param {Object} jem Reference to jem.
+    @param {Object} users The currently authenticated user info.
+    @param {Function} autoPlaceUnits The method used to auto place units.
+    @param {Function} createSocketURL The method used to create a socket URL.
+    @param {Function} appSet The method used to set parameters on the app.
+    @param {Boolean} committed Whether the model is already committed.
+    @param {Function} callback The function to be called once the deploy is
+      complete.
+    @param {String} model The name of the new model.
+    @param {String} credential The credentials to be used with the new model.
+    @param {String} cloud The cloud to deploy to.
+    @param {String} region The cloud region to deploy to.
+  */
+  utils.deploy = function(
+    env, jem, users, autoPlaceUnits, createSocketURL, appSet, committed,
+    callback, autoplace=true, model, credential, cloud, region) {
```

Comment:
Ah oops, `autoplace` is a flag for whether we should place unplaced units, `autoPlaceUnits` is the method for doing so.